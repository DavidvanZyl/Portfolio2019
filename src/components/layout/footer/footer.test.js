import React from "react"
import Footer from "./footer"

describe("Footer", () => {
  test("renders correctly", () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })
})
