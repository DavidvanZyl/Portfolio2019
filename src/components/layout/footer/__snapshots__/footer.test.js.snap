// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="footer__copy"
  >
    Â© 
    2019
    , David van Zyl
  </div>
  <_default>
    <div>
      <SocialButton
        title="Github"
        url="https://github.com/DavidvanZyl"
      />
      <SocialButton
        title="LinkedIn"
        url="https://www.linkedin.com/in/davidvzyl"
      />
    </div>
    <div>
      <SocialButton
        title="Medium"
        url="https://medium.com/"
      />
      <SocialButton
        title="Email me!"
        url="mailto:info@davidvanzyl.io"
      />
    </div>
  </_default>
</footer>
`;
